javascript/big-practice/src/scripts/utils/fetchAPI.js

```diff
@@ -1,89 +1,93 @@
 import URL_API from '../constants/apiUrl';
 
-const notesURL = `${URL_API.BASE_URL}${URL_API.NOTES_URL}`;
+const fetchAPI = {
```

Comment:
The fetchAPI will be used for all cases of the app. So we shouldn't define a special in this method. I mean, you should paste notesUrl as a param when calling any method in this file. And using common naming for results in each method.